scripts/es-integration-test.sh

```diff
@@ -103,30 +77,30 @@ bring_up_storage() {
   do
     echo "attempt $retry"
     if [ "${distro}" = "elasticsearch" ]; then
-      cid=$(setup_es "${version}")
+      compose_file=$(setup_es "${version}")
     elif [ "${distro}" == "opensearch" ]; then
-      cid=$(setup_opensearch "${version}")
+      compose_file=$(setup_opensearch "${version}")
     else
       echo "Unknown distribution $distro. Valid options are opensearch or elasticsearch"
       usage
     fi
-    wait_for_storage "${distro}" "http://localhost:9200" "${cid}"
+    wait_for_storage "${distro}" "http://localhost:9200" "${compose_file}"
```

Comment:
this line echoes in the log like this:
```
T19:25:41 wait_for_storage opensearch http://localhost:9200 '1.3.9: Pulling from opensearchproject/opensearch
```
i.e. instead of the compose file you're getting some other random output from `docker compose`. You may want to redirect stdout of docker compose command into stderr so that it does not mix up with your `echo ${compose_file}`

Reply:
now rather then taking it from` echo ${compose_file}` I am now create the variable in` bring_up_storage() `itself